spec/core_ext/logger_spec.rb

```diff
@@ -1,12 +1,15 @@
 require 'spec_helper'
 
-
 require 'logger'
 require 'awesomer_print/core_ext/logger'
 
 RSpec.describe 'AwesomerPrint logging extensions' do
   before(:all) do
-    @logger = Logger.new('/dev/null') rescue Logger.new('nul')
+    @logger = begin
```

Comment:
What's Rubocop's point here? It's literally the same, just uglier.

Reply:
The idea here is not not rescue form any error, but from a specific error to avoid false positives.

To this end, we should probably rescue from a more specific error here. 

```rb
Logger.new('/some/nonexistent/file')
# => Errno::ENOENT (No such file or directory @ rb_sysopen - /some/nonexistent/file)

begin
  Logger.new('/some/nonexistent/file')
rescue Errno::ENOENT
  puts "that didn't work"
end
# that didn't work
# => nil
```

> Style/RescueModifier: Avoid using rescue in its modifier form.

> The cop to check `rescue` in its modifier form is added for following reasons:
> 
>- The syntax of modifier form `rescue` can be misleading because it might led us to believe that `rescue` handles the given exception but it actually rescue all exceptions to return the given rescue block. In this case, value returned by handle_error or SomeException.
> 
>- Modifier form `rescue` would rescue all the exceptions. It would silently skip all exception or errors and handle the error. Example: If `NoMethodError` is raised, modifier form rescue would handle the exception.

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RescueModifier